app/app.html

```diff
@@ -5,7 +5,8 @@
   <link rel="stylesheet" href="../dist/css/patternfly.css">
   <link rel="stylesheet" href="../dist/css/patternfly-additions.css">
   <link rel="stylesheet" href="../dist/css/patternfly-webcomponents.css">
-  <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+  <script src="https://rawgit.com/webcomponents/webcomponentsjs/master/webcomponents-lite.js"></script>
```

Comment:
You may want to use webcomponents-loader.js in place of webcomponents-lite. It will more reflect the real world usage of web components